docs/extend/extend-cli/cli-extending-a-plugin.md

```diff
@@ -10,77 +11,85 @@ This tutorial demonstrates how to extend the plug-in that is bundled with this s
 3. Creating a command definition
 4. Creating a command handler
 
-We'll do this by using `@zowe/imperative` infrastructure to surface REST API data on our Zowe&trade; CLI plug-in.
+To do this, use the `@zowe/imperative` infrastructure to surface REST API data on our Zowe&trade; CLI plug-in.
 
-Specifically, we're going to show data from [this URI](https://jsonplaceholder.typicode.com/todos) by [Typicode](https://jsonplaceholder.typicode.com/).
-Typicode serves sample REST JSON data for testing purposes.
+Specifically, the tutorial shows data **[is the phrasing here correct?]** from [this URI](https://jsonplaceholder.typicode.com/todos) by [Typicode](https://jsonplaceholder.typicode.com/). Typicode serves sample REST JSON data for testing purposes.
```

Comment:
@adam-wolfe please respond

Reply:
The phrasing makes sense. It would be good to preface the list (starting at line 9) with the information in line 16 (describing what Typicode is). 

At it is now, it's not clear what is meant by "Creating a Typescript interface for the Typicode response data" until after reading line 16.